windows/installer/MozillaVPN.wxs

```diff
@@ -113,13 +113,6 @@
       </Component>
     </ComponentGroup>
 
-    <!--
-      Merge modules
-    -->
-    <DirectoryRef Id="MozillaVPNFolder">
```

Comment:
I am waaaay to close to a real windows PC to test this but afaik we require a VC_redist to be installed for the client to run. So if none are installed, (which is unlikey tbf) things might break, no ?

Reply:
My understanding is that from Windows 10 onwards microsoft has switched to a universal CRT that no longer requires a vcredist, and since the WIX installer goes out of its way to deny installation on earlier versions of Windows then this shouldn't be needed anymore.

But yes, much testing is required.

Reply:
Alas, I finally went and tested it and unfortunately it seems we really do need to install the MSVC redistributable somehow for the VPN client to run on Windows 10. Ugh. Back to the drawing board with this one.

Reply:
Okay, so, I really don't want to have to download the entire VS environment just to extract the MSM/VCRedist package, but I can't really find a good link to it on its own. The official word from Microsoft is that vcredists as MergeModules are [deprecated](https://learn.microsoft.com/en-us/cpp/windows/redistributing-components-by-using-merge-modules?view=msvc-170) and that we should be using the `vcredist_x64.exe` intstead, which should be version-agnostic.

However, this isn't a great solution either because of how the vcredist is generated... see [this thread](https://developercommunity.visualstudio.com/t/deprecation-of-crt-redist-msms/439217) for some complaints from the VS developer community.

Reply:
Oh no... that sounds like a familiar mess: https://developercommunity.visualstudio.com/t/MSMs-for-VCRT-from-VS2019-makes-MSI-pack/1143857